documentation/Functions.md

```diff
@@ -281,19 +281,32 @@ If the tutorial should be tested on the console environment, you have to specify
 ***
 
 ### npmInstall <a name="npmInstall"></a>
+npmInstall is used for installing packages from node package mangager, for example used in nodejs or angular projects.
 #### parameter 
-1. Path to the project where the dependencies from the package.json file are to be installed.
-2. Json-object: Name of a package, global or local installation, or array of npm arguments
-* (Optional) name of a package {"name": string }
-* (Optional) global or local installation. Default is local, therefore false {"global" : boolean }
-* (Optional) array of npm arguments {"args": string[]}
+You need to two parameters for this function.
+
+Example: npmInstall(1st-parameter, 2nd-parameter)
+
+1st-parameter: It is always needed. It is the path to the project where the dependencies from the package.json file are to be installed. It is of type string.
```

Comment:
I think we don't need to mention when a parameter is explicitly required. Just mention when a parameter is optional